src/Illuminate/Events/Dispatcher.php

```diff
@@ -156,19 +156,31 @@ public function flush($event)
      *
      * @param  object|string  $subscriber
      * @return void
+     *
+     * @throws \Illuminate\Contracts\Container\BindingResolutionException
```

Comment:
This must be added to the contract too.

Reply:
Added

Reply:
This method doesn't throws this exception, please remove it.